user-registration-app/src/app/views/registration-form/registration-form.component.ts

```diff
@@ -161,21 +161,28 @@ export class RegistrationFormComponent implements OnInit {
 
   ngOnInit() {
     this.route.data.subscribe(({ registration }) => {
-      this.registrationForm = registration;
-      this.registrationForm.firstHackathon = false;
-      this.registrationForm.eighteenBeforeEvent = null;
-      this.registrationForm.mlhcoc = null;
-      this.registrationForm.mlhdcp = null;
-      this.parsePhone(this.registrationForm.phone);
-      this.phoneNoUse = this.prettifiedPhone;
-      this.diet_restr = this.registrationForm.dietaryRestriction !== null;
+      this.registrationForm = new Registration();
```

Comment:
Is this needed if it does the same thing 9 lines above?

Reply:
see above, already changed and in production.